docker/docker-compose.yml

```diff
@@ -3,7 +3,16 @@ version: '2'
 services:
   conductor-server:
     environment:
-      - CONFIG_PROP=config.properties
+      - db=dynomite
+      - workflow_dynomite_cluster_hosts=dyno1:8102:us-east-1c
+      - workflow_dynomite_cluster_name=dyno1
+      - workflow_namespace_prefix=conductor
+      - workflow_namespace_queue_prefix=conductor_queues
+      - queues_dynomite_threads=10
+      - queues_dynomite_nonQuorum_port=22122
+      - workflow_elasticsearch_url=es:9300
+      - workflow_elasticsearch_index_name=conductor
```

Comment:
From my understanding these are `ConductorConfig` properties that are now being exposed as environment variables. This might be out of scope for this PR, but it seems unusual that env vars are lowecased. I have almost always seen them in caps.

I wonder if a better pattern for this is:

1. Expose the necessary configuration here as ENV vars
2. Pass and set these vars as ConductorConfig properties. For example - the env var `WORKFLOW_ELASTICSEARCH_INDEX_NAME` would map to CC's `workflow.elasticsearch.index.name`

Java is not my native language so I may be totally out of line here.